### PR TITLE
feat(motd): create MOTD based on day

### LINKED ISSUE #1: H10 MOTD enhancement
the `h10-motd` is installed on all hackerspace computers, but currently there is only one motd.  Needs more! Come up with a motd list for day of the year?  or find an API we can pull from with a few of our own.

https://github.com/timberline-secondary/hackerspace-control-repo/blob/production/site/profile/files/config/h10-motd.sh

### PR DESCRIPTION
**Pull request by punctuations**
Additions:
- cowsay
- custom cow for cowsay
- added cowsay to dependencies
- updated MOTD list

Dependencies:
- cowsay@latest

closes #1

### CODE REVIEW

File: site/profile/files/config/h10-motd.sh [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
-RED='\033[1;31m'
-NC='\033[0m'
+MOTDS=("Tetris is the GOAT." "Petition for morbius role!" "Hello, my name is Matthew Baker, I am currently trapped inside this computer, please send help I don't think Mr. C is sane." "Enjoy your time here!" "Today will be a day!" "Human children are lame" "We found the one piece. I won't say where." "Don't play 27 on the theme player" "Heresy" "Nothing bad is going to happen November 6, 2025" "2023" "Hello young scholar" "I hope the iPhone notch is gone" "idk" "Nutter Butter!" "What's that on the wall?" "I can see." "I love asparagus" "Fun fact: ANSI sucks." "I am alive" "<INSERT MOTD HERE>" "// TODO" "The original NES version of Tetris was technically a pirated port" "Tetris was the first video game played in space." "The tetris theme is actually a 19-th century Russian folk song" "Theoretically, using an algorithm it is possible to play one game of Tetris forever" "Dad jokes are the best jokes" "Stop! I'm trying to sleep." "Python? More like schmython" "where were you when club penguin is kill?" "https://youtu.be/GtL1huin9EE")
```
Comment (tylerecouture) [inline site/profile/files/config/h10-motd.sh]: Can you put this in a text file instead, with each on its own line, then read that file?

File: site/profile/manifests/base.pp [hunk 2]
```diff
@@ -22,6 +22,10 @@
     ensure  => latest,
   }
 
+  package { 'cowsay':
+    ensure => latest,
+  }
+
```
Comment (tylerecouture) [inline site/profile/manifests/base.pp:28]: You'll also need to add the puppet code to install the custom cow file. See the code below for how the motd.sh is "saved" to each of the clients. Maybe save it in /etc/motd/

And the motd text list too I request above

**Review by tylerecouture — CHANGES_REQUESTED** [review]

File: site/profile/files/config/h10-motd.sh (hunk 1 above)
Comment (punctuations) [reply site/profile/files/config/h10-motd.sh]: fixed as per [dec3a29](https://github.com/timberline-secondary/hackerspace-control-repo/pull/2/commits/dec3a2992c8383f32609f24f8aaf7230e905d521)

File: site/profile/manifests/base.pp (hunk 2 above)
Comment (punctuations) [reply site/profile/manifests/base.pp:28]: fixed as per [93dfac7](https://github.com/timberline-secondary/hackerspace-control-repo/pull/2/commits/93dfac7c92e88cc42c5b003aca0eb6b7ebf0d24b)

**Pull request merged**